Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -593,6 +593,33 @@ def audit_github_repository
       new_formula_problem "GitHub repository too new (<30 days old)"
     end
 
+    def audit_gitlab_repository
```

Comment:
Can this share any code with e.g. `audit_github_repository`? It looks like it might be similar except for the:
- regex
- url
- metadata format

Reply:
it's now sharing code via the `get_repo_data` function